static/maintenance.html

```diff
@@ -25,7 +25,7 @@
         <h1 style="text-align: center;">Undergoing Maintenance</h1>
         <p style="text-align: center;">This application is currently undergoing maintenance. Please wait a few minutes and try again.</p>
 
-        {# JQuery 2.x not supported by <= IE8, so we are loading JQuery 1.x for it #}
+        <!-- JQuery 2.x not supported by <= IE8, so we are loading JQuery 1.x for it -->
```

Comment:
@mabuaisha This line exists in many other HTML files such as 400.html, 503.html, etc...

Please apply what you did here to all those files.